GlyssenEngineTests/Quote/QuoteParserTests.cs

```diff
@@ -5370,6 +5370,28 @@ public void Parse_MultipleInterruptionsInMiddleOfMultiBlockQuote_MultiBlockQuote
 			Assert.AreEqual(CharacterVerse.kScriptureCharacter, output[5].CharacterId);
 		}
 		#endregion
+
+		[TestCase(19, 10, "6", "narrator-PSA")]
+		[TestCase(40, 7, "8", "Jesus")]
+		public void GetCharacters_ReferenceTextIsNull_ReturnsOnlyOneCharacterForVerse(int book, int chapter, string verse, string expectedCharacter)
+		{
+			var cvRepo = new ParserCharacterRepository(ControlCharacterVerseData.Singleton, null);
+
+			var character = cvRepo.GetCharacters(book, chapter, new Verse(verse));
+
+			Assert.AreEqual(expectedCharacter, character.First().ToString());
+		}
+
+		[Test]
+		public void GetCharacters_ReferenceTextIsNull_ReturnsMultipleCharactersForVerse()
+		{
+			var cvRepo = new ParserCharacterRepository(ControlCharacterVerseData.Singleton, null);
+
+			var character = cvRepo.GetCharacters(45, 2, new Verse("15"));
+
+			Assert.AreEqual("Gentiles", character.First().ToString());
```

Comment:
First and Last don't really make sense. GetCharacters returns a HashSet, so it is not really ordered. You're just getting lucky. You probably just want to check that the total number is as expected and that the set contains the known values. (FWIW, I found this a bit confusing because there really shouldn't be any characters in ROM 2:15, but the test data has them there, so I guess it's okay.)

Reply:
Once I moved them to their own class and file I was getting different values and you are right Romans 2:15 didn't have any characters.  Strange.

Reply:
See if my test is ok for multiple numbers.